zotero2readwise/zotero.py

```diff
@@ -41,8 +41,11 @@ def __post_init__(self):
         # Sample {'dc:relation': ['http://zotero.org/users/123/items/ABC', 'http://zotero.org/users/123/items/DEF']}
         if self.relations:
             self.relations = self.relations.get("dc:relation")
-
-        self.creators = ", ".join(self.creators) if self.creators else None
+        
+        if self.creators:
+            while ", ".join(self.creators) >= 1024 - len(", et al."):
+                self.creators = self.creators[:len(self.creators) - 1]
+            self.creators = ", ".join(self.creators) + ", et al."
```

Comment:
@noeleont I believe you're missing `len(()` in while condition

Reply:
While I haven't ran the following, the following refactoring might be more efficient. What's your thought?

```python
if self.creators:
    creators_str = ", ".join(self.creators)
    et_al = ", et al."
    max_length = 1024 - len(et_al)
    # Only proceed if the string is too long
    if len(creators_str) > max_length:
        # Find the last creator that fits within the limit
        while len(creators_str) > max_length and self.creators:
            self.creators.pop()
            creators_str = ", ".join(self.creators)
        # Add 'et al.' only if there are remaining creators
        if self.creators:
            self.creators = creators_str + et_al
```

Reply:
I merged our approaches, can you have a look?